### PR TITLE
BlogApp: controllers and routes

### PR DESCRIPTION
**Pull request by V0pe**
## Controllers

In this milestone, I did the following:

- [ ] Add the controllers
- [ ] Add the methods and views
- [ ] Add the routes

### CODE REVIEW

**Review by OLIPLICHE — COMMENTED** [review]
Hi @V0pe, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

I advise you to delete the live demo section if you decide not to deploy the application. Kindly do not use a fake live demo link. This is because it is part of the general requirement, and to make your work more professional. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def user; end
+
+  def posts; end
+
+  def post; end
+end
```
Comment (uwadonat) [inline app/controllers/users_controller.rb]: - [ ] Apart from this users_controller, let's also add `posts_controller` with `index` and `show` actions as requested in the project requirement.

File: config/routes.rb [hunk 2]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get 'users/index'
+  get '/users', to: "users#index"
+  get '/users/:id/', to: "users#user"
+  get '/users/:id/posts', to: "users#posts"
+  get '/users/:id/posts/:id', to: "users#post"
+  root 'users#index'
```
Comment (uwadonat) [inline config/routes.rb]: - [ ] let's not expose unnecessary endpoints. you can just write your routes as follow:
````
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @V0pe,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Professional readme file ✔️
- No linter errors ✔️
- Correct workflow was used ✔️
almost


## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Victor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 2 above)
Comment (V0pe) [reply config/routes.rb]: @uwadonat Thanks

**Pull request merged**